src/HostManager.cpp

```diff
@@ -38,39 +38,34 @@
 #include "post_guard.h"
 
 
-bool HostManager::deleteHost( QString hostname )
+bool HostManager::deleteHost(QString hostname)
 {
     qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
@SlySven agree on removing the mutex debug lines you've put in - they've served their purpose and I haven't seen any issues stemming from that area. Time to get rid of them.